### ISSUE TITLE
Form elements must have labels

### ISSUE THREAD

**Issue by bmbc-dev-detec** [LocalGovIMS/localgov-ims#331]
Multiple fields across the site's forms have no label associated with them.

Effective form labels are required to make forms accessible. The purpose of form elements such as checkboxes, radio buttons, input fields, etcetera, is often apparent to sighted users, even if the form element is not programmatically labelled. Screen readers users require useful form labels to identify form fields. Adding a label to all form elements eliminates ambiguity and contributes to a more accessible product.

**Comment by bmbc-dev-detec** [comment]
Ready for retesting after completion of #416

**Comment by bmbc-dev-detec** [comment]
Closing. Will add any problem found after retesting as new issues
